lib/daos/driverDao.js

```diff
@@ -0,0 +1,64 @@
+import { models } from '@models';
+import { badImplementation } from '@utils/responseInterceptors';
+import sequelize from 'sequelize';
+
+export const findOneUser = async () => models.users.findOne();
```

Comment:
Can't we remove this?

Reply:
If I remove this. We will have to remove it from many other places so I didn't removed this.